composer/algorithms/colout/colout.py

```diff
@@ -173,16 +201,17 @@ def _apply_sample(self, state: State) -> None:
                 textwrap.dedent(f"""\
                 To use {type(self).__name__}, the dataset must be a
                 {VisionDataset.__qualname__}, not {type(dataset).__name__}"""))
-        add_vision_dataset_transform(dataset, transform, is_tensor_transform=False)
+        add_vision_dataset_transform(dataset, transform, is_tensor_transform=False, is_target_transformed=True)
         self._transformed_datasets.add(dataset)
 
     def _apply_batch(self, state: State) -> None:
         """Transform a batch of images using the ColOut augmentation."""
-        inputs, labels = state.batch_pair
-        assert isinstance(inputs, Tensor), "Multiple Tensors not supported yet for ColOut"
-        new_inputs = colout_batch(inputs, p_row=self.p_row, p_col=self.p_col)
+        inputs, targets = state.batch_pair
+        assert isinstance(inputs, Tensor) and isinstance(targets, Tensor), \
+            "Multiple tensors not supported for this method yet."
```

Comment:
This could be confusing as the method does support two tensors now. Maybe split this into separate checks for inputs and targets, or reword to specify that both inputs and targets should be single tensors?